tests/e2e/specs/price-benchmark/price-benchmark-suggestions.test.js

```diff
@@ -97,17 +91,44 @@ test.describe( 'Price Benchmark Page', () => {
 	} );
 
 	test.describe( 'Price Benchmark Suggestions Functionality', () => {
-		test( 'Shows no results if there is no data', async () => {
+		test.beforeEach( async () => {
 			await priceBenchmarkPage.goto();
+		} );
 
+		test( 'Shows no results if there is no data', async () => {
 			await priceBenchmarkPage.fulfillPriceBenchmarkSuggestions( [] );
 
-			await expect( page.getByText( 'No results' ) ).toBeVisible();
+			const emptyStateNotice = page.locator(
+				'.gla-price-benchmark__empty-metrics'
+			);
+
+			await expect( emptyStateNotice ).toBeVisible();
+			await expect( emptyStateNotice ).toContainText(
+				'You do not have any sale price suggestions at this moment.'
+			);
+			await expect( emptyStateNotice ).toContainText(
+				'Find out if you meet all eligibility criteria to receive suggestions in the future.'
+			);
 		} );
 
-		test( 'Shows 10 results per page by default', async () => {
-			await priceBenchmarkPage.goto();
+		test.only( 'Shows empty state notice when Market Insights is not enabled for the account.', async () => {
+			await priceBenchmarkPage.fulfillPriceBenchmarkSuggestions(
```

Comment:
While this would work, since you're returning a `403` error code, the response is not an empty array. The response should be similar to the error response here: https://github.com/woocommerce/google-listings-and-ads/issues/2852